OasisDEX/oasis-borrow
# 🧿 Split get Ajna position observable

# 🧿 Split get Ajna position observable

Since library requires quote price, token prices has to loaded first before position is loaded, main observable had to be split into two separated ones. This PR does not pass quote price to library (library is not updated so it doesn't require it) but allows to do it in easy way.
  
## Changes 👷‍♀️

- Remove `dpmPositionData$` observable dependency from `ajnaPosition$` and replaced it with `dpmPositionData` interface,
- created `getStaticDpmPositionData$` fakeish observable,
- updated `AjnaProductController` observable order and dependencies.
  
## How to test 🧪

Try both open and manage flow and see if empty position and existing positions are loading correctly.
